### PR TITLE
Bump eks-operator chart to 1.1.1-rc4

### PR DESCRIPTION
**Pull request by thedadams**
Issue:
https://github.com/rancher/rancher/issues/29066

### CODE REVIEW

**Comment by jiaqiluo** [comment]
@thedadams  Can you update your PR following those steps:
- change the url in `packages/rancher-eks-operator-crd/package.yaml` and `packages/rancher-eks-operator/package.yaml` to the ones for `1.1.1-rc4`
- delete `rancher-eks-operator-2.0.0+up1.1.1-rc2` and `rancher-eks-operator-crd-2.0.0+up1.1.1-rc2` from the dir charts/ and assests/
- delete `index.yaml`
- run `make charts` 


let me know if you have any question.

File: packages/rancher-eks-operator/package.yaml [hunk 1]
```diff
@@ -1,2 +1,2 @@
-url: https://github.com/rancher/eks-operator/releases/download/v1.1.1-rc2/rancher-eks-operator-1.1.1-rc2.tgz
-version: 2.0.0
```
Comment (jiaqiluo) [inline packages/rancher-eks-operator/package.yaml:2]: no need to change the version.

File: packages/rancher-eks-operator-crd/package.yaml [hunk 2]
```diff
@@ -1,2 +1,2 @@
-url: https://github.com/rancher/eks-operator/releases/download/v1.1.1-rc2/rancher-eks-operator-crd-1.1.1-rc2.tgz
-version: 2.0.0
```
Comment (jiaqiluo) [inline packages/rancher-eks-operator-crd/package.yaml:2]: no need to change the version.

**Review by jiaqiluo — CHANGES_REQUESTED** [review]

**Review by jiaqiluo — APPROVED** [review]
LGTM

**Review by jiaqiluo — APPROVED** [review]

**Review by cmurphy — APPROVED** [review]

**Review by cbron — APPROVED** [review]

**Pull request merged**